src/components/schedule-page/import-buttons/import-buttons.component.tsx

```diff
@@ -71,21 +77,18 @@ export function ImportButtonsComponent(): JSX.Element {
             setOpen(false);
           }}
         >
-          <ButtonGroup orientation="vertical">
-            <Button onClick={(): void => fileUpload.current?.click()}>
-              Wczytaj
-              <input
-                ref={fileUpload}
-                id="file-input"
-                data-cy="file-input"
-                onChange={(event): void => handleImport(event)}
-                style={{ display: "none" }}
-                type="file"
-                accept=".xlsx"
-              />
-            </Button>
-            <Button onClick={(): void => handleExport()}>Zapisz jako...</Button>
-          </ButtonGroup>
+          <div>
+            {DropdownButtonsComponent(btnData)}
```

Comment:
Lepiej tworzyć komponenty za pomocą takiej składni.:
```
<DropdownButtonComponent buttons={btnData}>
```
Z tym, że wtedy potrzeba by było też zmienić nieco sygnatuę tego komponentu, co ty teraz używasz: zamiast 
```
 export default function DropdownButtonsComponent(btnData: ButtonData[])
```
Będzie coś w stylu:
```
interface Options {
  buttons: ButtonData[]
}
export default function DropdownButtonsComponent({buttons}: Options) {
.....
```
PS.
zwróć uwagę na klamry które pojawiły się w parametrach


PS.PS.
To jest dekonstrukcja słownika, która pozwala ci dostać klucz z słownika i odrazu przypisać go do zmiennej. 
Czyli, mając 
```
const a = {key1: 1, key2: 3}
```
Zamiast pisać coś w stylu
```
const key1 = a.key1;
```
Możemy napisać
```
const { key1 } = a

```
PS.PS.PS
W przypadku, gdy użyjemy `Component()` zamiast `<Component/>`, może pojawić się taki error

https://medium.com/@jonchurch/how-to-fix-react-error-rendered-fewer-hooks-than-expected-e6a378985d3c
![image](https://user-images.githubusercontent.com/21079319/101244898-685b9d80-3709-11eb-991e-8fe5edd8b46c.png)

Reply:
Dzięki za uwagę, już zmieniłam :)